src/components/Sections/GroupedSectionsView.tsx

```diff
@@ -23,6 +24,7 @@ export const GroupedSectionView = ({
   ): void => {
     navigate(`categories/${type}/${name}`)
   }
+  const isServicesView = sections.find(section => section.type === 'category')
```

Comment:
I realised that it's a bit confusing to use services naming for connectors (I know it was done before you). Maybe in this case it can appropriate to use `hasCategorySection` but I'm not sure of the naming 🤔

Reply:
I think in this case I really want to be explicit that I'm targetting "Services" (indeed in home that's how konnectors are called everywhere)